recipe/meta.yaml

```
@@ -1,33 +1,51 @@
-{% set version = "301" %}
+{% set version = "302" %}
 
 package:
   name: pywin32
   version: {{ version }}
 
 source:
   url: https://github.com/mhammond/pywin32/archive/b{{ version }}.tar.gz
-  sha256: 5366613c366c66b521a2e79780dcdf347310bd2b75022e424df320760931999c
+  sha256: 47fd53c0561ff3019043e86880597aacbd736910d8add485117d8ad055441927
   patches:
     - disable_exchange_modules.patch
     - 0003-remove-PATH-pth-hack.patch
 
 build:
-  number: 0
+  number: 2
```

Can we do the same as #37 and add
```suggestion
  number: 2
  skip: True            # [(build_script == "pip" and not win) or (build_script == "noop" and win)]
  noarch: generic   # [build_script == "noop" ]
```
along with cbc.yaml?

---

I don't actually trust that mechanism enough.  When windows-arm64 rolls around i'd prefer to not have packages for that platform

---

the other part is we needed the fake windows ones too, so there is that part

---

With this approach there is no possible way for windows to even consider installing a dummy noarch version

---

> I don't actually trust that mechanism enough. When windows-arm64 rolls around i'd prefer to not have packages for that platform

The proposed change would go along with the `requirements: { run: [ __unix ] }` from gh-37. I.e., a future win-arm64 shouldn't be able to install those builds.

> the other part is we needed the fake windows ones too, so there is that part

I don't understand. Do you mean for PyPy?

----
Just to reiterate for my (and possibly others') understanding:
If PyPy doesn't need `pywin32` then the minimum we'd need is
1. non-empty builds with `python_abi=*_cp*` run deps,
2. an empty build with `python_abi=*_pypy*_pp*` run dep on Windows,
3. an empty build with `__unix` run dep that is `noarch: generic`.

With the current state of this PR we'd just produce surplus empty packages for point 2 and 3. But at least we don't build anything falls outside the requirements (i.e., no empty builds for Win+CPython and no non-empty builds for the other cases.

---

But a future win-arm64 will still be able see them by using `conda search pywin32` which is what i'm trying to avoid here